src/features/locators/locators.selectors.ts

```diff
@@ -18,8 +16,7 @@ export const selectLocatorById = createSelector(selectById, (_item?: Locator) =>
       ..._item,
       locator: {
         ..._item.locator,
-        output:
-          _item.locatorType === LocatorType.cssSelector ? _item.locator.cssSelector : getXPathByPriority(_item.locator),
+        output: _item.locator.output ?? getLocator(_item.locator, _item.locatorType),
```

Comment:
do we have a case when we save smth to `output`?
so far it's a synthetic field and doesn't saved in state

Reply:
no, fixed

Reply:
good, thank you!